src/fal/cli/args.py

```diff
@@ -130,20 +128,16 @@ def _add_experimental_python_models_option(parser: argparse.ArgumentParser):
         "--experimental-models",
         dest="experimental_python_models",
         action="store_true",
-        help="""
-        Search Python files in source paths and treat as pure Python models. Implies --experimental-flow.
-        """,
+        help="DEPRECATED: no-op",
```

Comment:
NIT: maybe just hide it, with `help=argparse.SUPPRESS`?

Reply:
Not sure... any strong opinions, @turbo1912 ?